### ISSUE TITLE
Apexcharts not displaying %age on pie charts for wxWebView

### ISSUE THREAD

**Issue by whalley** [moneymanagerex/moneymanagerex#2823]
MMEX version:
 - [X] 1.3.7
 - [ ] 1.3.6

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [ ] Windows
 - [X] Mac OSX
 - [ ] Linux 

# Description of the bug

Issue with ApexCharts presentation of %age on Pie charts. On MacOs WxWebView uses the underlying MacOs WebView class (which is actually deprecated) - not sure if this is the issue. Works fine if same HTML is displayed in other browserson MacOs and Windows.

# Reproduction

Is the bug reproducible? 
 - [X] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Will raise this with ApexCharts developers.....

@vomikan Please assign to me.

**Comment by whalley** [comment]
Looks like latest WxWidgets now has support form WkWebView - Will set aside some time to build and test --- This may fix the issue.

**Comment by whalley** [comment]
Have built against the latest WkWebView (after making a few tweaks to support it), but it still does not display the segment % values. Will raise query with WxWidgets team also.

**Comment by whalley** [comment]
Narrowed the problem to the user agent detection. It does not correctly identify non-Safari webkit. Have made an edit and used the modified apexcharts.js file until I can get the changes into the main apexcharts code base.

https://github.com/whalley/moneymanagerex/commit/2edab3e8ccfb8bf444e2cb9bfb5dd4b71c6e07c9

**Comment by vomikan** [comment]
I've merged it. Let me know if this is not correct.

**Comment by whalley** [comment]
Thanks for merging, this will be fine until we get formal fix. Cna we just tag this issue with 'on hold' for now. Don't want to close until Apex Charts team fix it.
